Contents/Code/webapp.py

```diff
@@ -328,6 +331,10 @@ def home():
                 if not os.path.isdir(theme_upload_path) or not os.listdir(theme_upload_path):
```

Comment:
I'm working on that, I found a way to see whether the currently used theme has been uploaded (i.e. through the API) or is coming from a manually put theme.mp3 file. I'll add that after #208 is merged since bits of mechanics are shared